aspnet/mvc/views/partial.rst

```diff
@@ -1,8 +1,128 @@
-.. include:: /../common/stub-topic.txt
+Partial Views
+=============
 
-|stub-icon| Partial Views
-=========================
+By `Steve Smith`_
+
+ASP.NET Core MVC supports partial views, which are useful when you have reusable parts of web pages you want to share between different views.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/mvc/views/partial/sample>`__
+
+What are Partial Views?
+-----------------------
+
+A partial view is a view that is rendered within another view. Typically they render a block of HTML that can be safely inserted within one or more parent views. The partial view is built up at runtime by the :doc:`razor engine <razor>` within the parent view that contains it. Like views, partial views use the *.cshtml* file extension.
+
+.. note:: If you're coming from an ASP.NET Web Forms background, partial views are similar to `user controls <https://msdn.microsoft.com/en-us/library/y6wb1a0e.aspx>`_.
```

Comment:
I don't think @dougbu  would approve of that comparison. @danroth27 typically doesn't like historical comparisons. I think that might get us in trouble as there are more differences than similarities.